core/aws-core/src/main/java/software/amazon/awssdk/awscore/AwsServiceClientConfiguration.java

```diff
@@ -76,15 +76,6 @@ public interface Builder extends SdkServiceClientConfiguration.Builder {
          */
         Builder region(Region region);
 
-        @Override
-        Builder overrideConfiguration(ClientOverrideConfiguration clientOverrideConfiguration);
```

Comment:
Why are we removing these?

Reply:
Don't they refine the return type?

Reply:
It would be backwards-incompatible (and imo, incorrect) to remove them. They have a different return type.

Reply:
Yeah, talked with @cenedhryn offline and there were added back